### PR TITLE
Fixing output and inttests

### PR DESCRIPTION
**Pull request by marians**
Ping @zyndiecate or @ZeissS for a quick bullshit check

### CODE REVIEW

File: int-tests/100-create.t [hunk 1]
```diff
@@ -3,9 +3,9 @@ Create:
   $ if [ -z ${ALL} ]; then exit 80; fi # Skip unless ALL is non-empty
   $ swarm -q delete --yes beep 2>&1 > /dev/null
   $ swarm create ${TESTDIR}/apps/beep/swarm.json
-  Creating 'beep' in the 'cli-testbot/dev' environment...
+  Creating 'beep' in the 'cli-testbot/dev' environment\xe2\x80\xa6 (esc)
```
Comment (ZeissS) [inline int-tests/100-create.t]: that is very weird Oo

**Comment by ZeissS** [comment]
I am confused by those escaped character, the rest is fine.

File: int-tests/100-create.t (hunk 1 above)
Comment (marians) [inline int-tests/100-create.t]: This is the right notation for the utf8 ... character we use there.

**Comment by marians** [comment]
Thanks @ZeissS

**Pull request merged**